docs/sql-migration-guide.md

```diff
@@ -91,6 +91,8 @@ license: |
 
   - In Spark 3.2, `CREATE TABLE AS SELECT` with non-empty `LOCATION` will throw `AnalysisException`. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowNonEmptyLocationInCTAS` to `true`.
 
+  - In Spark 3.2, the special datetime values such as `epoch`, `today`, `yesterday`, `tomorrow` and `now` are supported in typed literals only, for instance `select timestamp'now'`. In Spark 3.1 and earlier, such special values are supported in any casts of strings to dates/timestamps. To restore the behavior before Spark 3.2, you should preprocess string columns and convert the strings to desired timestamps explicitly using UDF for instance.
```

Comment:
In Spark 3.2, ~the~ special datetime values..... in typed literals only, for instance **(add',')** `select timestamp'now'`. In Spark 3.1 and ~earlier~ (3.0?)

Reply:
@yaooqinn What do you mean by: for instance **(add',')** `select timestamp'now'`. I didn't get the problem. BTW, you could use the suggestion feature, so, I would just commit your suggestions.

Reply:
Hi @MaxGekk, thanks for your suggestion. 

I think if users need to preprocess the data, we may not call it as`To restore the behavior before Spark 3.2`

Reply:
What do you propose? How about `To have the behavior before Spark 3.2 ...`

Reply:
How about "to keep these special values as datetimes in Spark 3.1 and 3.0, you need to match them manually, e.g. `if(c in ('now', 'today'), current_date() else c)`.

I think it's better to suggest user use builtin functions than UDFs

Reply:
I agree that builtin functions is better suggestion. Let's me update this.